### PR TITLE
Card back buttons routes to the previous card.

### PR DESCRIPTION
**Pull request by sandroroux**
Closes catalpainternational/asteroid#108.

### CODE REVIEW

File: src/riot/Lesson/LessonContent.riot.html [hunk 1]
```diff
@@ -91,7 +91,7 @@
             },
 
             onBeforeMount(props, state) {
-                state.cardIdx = props.isLessonPage ? getLessonCardIdx() : '0';
+                state.cardIdx = props.isLessonPage ? getLessonCardIdx() : "0";
```
Comment (PeteCoward) [inline src/riot/Lesson/LessonContent.riot.html:94]: ugh.. we are storing `cardIdx as a string :)

Comment (sandroroux) [reply src/riot/Lesson/LessonContent.riot.html:94]: Indeed. Would you like that to change in this PR?

Comment (sandroroux) [reply src/riot/Lesson/LessonContent.riot.html]: @PeteCoward I redirected this fix into RC-1.1. This change doesn't exist anymore.

**Comment by sandroroux** [comment]
@PeteCoward @katevoss I directed this bug fix into RC-1.1. Do y'all mind giving it another look?

**Comment by PeteCoward** [comment]
Incidentally I recently learned that we can use history.pushState and the popstate event for forward back history tracked changes which don't affect the url. Might be useful at some point

**Pull request merged**